private-templates-service/client/src/components/Dashboard/Dashboard.tsx

```diff
@@ -22,26 +27,31 @@ const mapDispatchToProps = (dispatch: any) => {
   return {
     setPage: (currentPageTitle: string) => {
       dispatch(setPage(currentPageTitle));
-    }
-  };
-};
+    },
+    getTemplates: () => { dispatch(getAllTemplates()) }
+  }
+}
 
 interface State {
   isPreviewOpen: boolean;
 }
 
 interface Props {
   isAuthenticated: boolean;
+  user?: UserType;
```

Comment:
why is `user `optional?

Reply:
Just noticed user is not needed on the dashboard page. I will remove them :)